### ISSUE TITLE
Build: Same page links to sections are not accessible with keyboard

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#63]
URL: https://hcdev.openplus.ca/en/node/1109/build#Section1 (click on Build for any Recall or Alert from Dashboard)
User Agent: Chrome, Firefox
Language: Both

Issue Description: User is not able to tab to the sections 1-6 on the right hand side of the page which are anchor links within the same page. Focus goes to Workflow link and then jumps to the footer of the page.

Steps to reproduce: Keyboard tabbing on the page.

Expected results: User should be able to tab to these.

![image](https://user-images.githubusercontent.com/14280062/129077577-154202db-a6c4-45ea-8d97-57e5aecc0d39.png)

**Comment by davidgotlieb** [comment]
his is fixed, replaced tocify with custom a block ﻿https://hcdev.openplus.ca/en/block/16?destination=/en/admin/structure/block/block-content﻿.

**Comment by qanitah** [comment]
Fixed
